linera-views/src/test_utils/mod.rs

```diff
@@ -306,27 +306,28 @@ pub async fn run_reads<S: LocalKeyValueStore>(store: S, key_values: Vec<(Vec<u8>
             }
         }
         let mut test_exists = Vec::new();
+        let mut values_single_read = Vec::new();
         for key in &keys {
             test_exists.push(store.contains_key(key).await.unwrap());
+            values_single_read.push(store.read_value_bytes(key).await.unwrap());
         }
         let values_read = store.read_multi_values_bytes(keys).await.unwrap();
         assert_eq!(values, values_read);
+        assert_eq!(values, values_single_read);
         let values_read_stat = values_read.iter().map(|x| x.is_some()).collect::<Vec<_>>();
         assert_eq!(values_read_stat, test_exists);
     }
 }
 
-fn get_random_key_values1(len_value: usize) -> Vec<(Vec<u8>, Vec<u8>)> {
+fn get_random_key_values1(n: usize, len_value: usize) -> Vec<(Vec<u8>, Vec<u8>)> {
```

Comment:
What is `n` for?

Reply:
The number of entries in the vector. It has been renamed to `num_entries`.